### ISSUE TITLE
During installation, make env installation fails at step 5

### ISSUE THREAD

**Issue by dg-deus** [f4pga/prjxray#2271]
I am trying to install it on Ubuntu 22.04.3 LTS virtualized with vmware workstation.
However, I keep having the following error.

The commands I used for installation are the below.

```
git clone https://github.com/f4pga/prjxray.git
git submodule update --init --recursive

sudo apt-get install cmake

# To get around "No CMAKE_CXX_COMPILER" error during build
sudo apt update
sudo apt upgrade
sudo apt install build-essential

make build

# Step 5
sudo apt-get install virtualenv python3 python3-pip python3-virtualenv python3-yam
sudo apt install python3.10-venv
make env
```

Then, I got the following error message.

```
python3 -mvenv env
# Install project dependencies
if [ -e env/bin/activate ]; then . env/bin/activate; fi; source utils/environment.python.sh; python -mpip install -r requirements.txt
Obtaining file:///home/user/prjxray/third_party/fasm (from -r requirements.txt (line 19))
  Installing build dependencies ... done
  Checking if build backend supports build_editable ... done
  Getting requirements to build editable ... done
  Preparing editable metadata (pyproject.toml) ... done
Obtaining file:///home/user/prjxray/third_party/python-sdf-timing (from -r requirements.txt (line 20))
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [34 lines of output]
      /home/user/prjxray/env/lib/python3.10/site-packages/setuptools/installer.py:27: SetuptoolsDeprecationWarning: setuptools.installer is deprecated. Requirements should be satisfied by a PEP 517 installer.
        warnings.warn(
      WARNING: The wheel package is not available.
      WARNING: The wheel package is not available.
      WARNING: The wheel package is not available.
      WARNING: The wheel package is not available.
      WARNING: The wheel package is not available.
      WARNING: The wheel package is not available.
      WARNING: The wheel package is not available.
      WARNING: The wheel package is not available.
      WARNING: The wheel package is not available.
      WARNING: The wheel package is not available.
      WARNING: The wheel package is not available.
      WARNING: The wheel package is not available.
      WARNING: The wheel package is not available.
      WARNING: The wheel package is not available.
      WARNING: The wheel package is not available.
      WARNING: The wheel package is not available.
      WARNING: The wheel package is not available.
      WARNING: The wheel package is not available.
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/home/user/prjxray/third_party/python-sdf-timing/setup.py", line 55, in <module>
          setuptools.setup(
        File "/home/user/prjxray/env/lib/python3.10/site-packages/setuptools/__init__.py", line 152, in setup
          _install_setup_requires(attrs)
        File "/home/user/prjxray/env/lib/python3.10/site-packages/setuptools/__init__.py", line 147, in _install_setup_requires
          dist.fetch_build_eggs(dist.setup_requires)
        File "/home/user/prjxray/env/lib/python3.10/site-packages/setuptools/dist.py", line 812, in fetch_build_eggs
          resolved_dists = pkg_resources.working_set.resolve(
        File "/home/user/prjxray/env/lib/python3.10/site-packages/pkg_resources/__init__.py", line 782, in resolve
          raise VersionConflict(dist, req).with_context(dependent_req)
      pkg_resources.ContextualVersionConflict: (platformdirs 4.0.0 (/home/user/prjxray/third_party/python-sdf-timing/.eggs/platformdirs-4.0.0-py3.10.egg), Requirement.parse('platformdirs<4,>=3.9.1'), {'virtualenv'})
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
make: *** [Makefile:27: env] Error 1
```

Has been something changed to python-sdf-timing package?

Thank you.

**Comment by dg-deus** [comment]
I got it fixed by adding `platformdirs==3.10.0` at prjxray/third_party/python-sdf-timing/requirements.txt

**Comment by dg-deus** [comment]
I got it fixed by adding platformdirs==3.10.0 at prjxray/third_party/python-sdf-timing/requirements.txt
